src/pages/workspaces/workspace/modals/FindWorkflowModal.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { expect } from '@storybook/test';
```

Comment:
The rogue `expect` has appeared again!

Reply:
Lol I am not sure how it keeps appearing. Will remove it! Thank you!